README.md

```diff
@@ -1,5 +1,11 @@
 [![Appveyor build status (Windows)](https://ci.appveyor.com/api/projects/status/xfllw9vkp3lj4l0v/branch/main?svg=true)](https://ci.appveyor.com/project/ispc/ispc/branch/main)
-
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/9395/badge)](https://www.bestpractices.dev/projects/9395)
```

Comment:
Let's create a separate section for security badges, otherwise it messes things up.

For the "core" section with build we might add a license badge to make build not look lonely:
```
[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
```

Reply:
OK!